uniffi_aries_vcx/demo/app/src/main/java/org/hyperledger/ariesvcx/AppDemoController.kt

```diff
@@ -110,4 +121,45 @@ class AppDemoController : ViewModel() {
     fun subscribeToConnectionComplete(onComplete: (connection: Connection) -> Unit) {
         onConnectionComplete = onComplete
     }
+
+    fun subscribeToShowDialog(onShowDialog: () -> Unit) {
+        onOfferReceived = onShowDialog
+    }
+
+    fun processOfferRequest() {
+        holder?.prepareCredentialRequest(profile!!, "4xE68b6S5VRFrKMMG1U95M")
+        val message = holder?.getMsgCredentialRequest()
+        connection?.sendMessage(profile!!, message!!)
+
+        _state.update { it.copy(offerReceived = false) }
+    }
```

Comment:
@gmulhearn the credential is not accepted here (atm I am sort of forcing the credential to be accepted).